packages/docutils/lib/cli/config.ts

```diff
@@ -46,9 +45,9 @@ export async function findConfig(argv: string[] = hideBin(process.argv)) {
 
   // if --verbose is used, set the log level to debug.
   // otherwise use --log-level or the default.
-  let logLevel: LogLevel | keyof typeof LogLevelMap;
+  let logLevel: keyof typeof LogLevelMap;
   if (preArgs.verbose) {
-    logLevel = LogLevel.Debug;
+    logLevel = 'debug';
```

Comment:
can we use LogLevels.debug ?

Reply:
`logLevel` has the type `"silent" | "error" | "warn" | "info" | "debug"`, but `LogLevels.debug` is `number` . Similarly, `LogTypes.debug` is `Partial<LogObject>`.